src/Core/tests/DeviceTests/Handlers/HandlerTestBase.Android.cs

```diff
@@ -15,5 +16,8 @@ protected THandler CreateHandler(IView view)
 
 			return handler;
 		}
+
+		protected string GetAutomationId(IViewHandler viewHandler) =>
```

Comment:
I know it is not this PR, but the `Activator.CreateInstance` above can also make use of the `new()` generic constraint and then we can new it up. And compiler safe.